src/components/forms/CorpusForm.tsx

```diff
@@ -57,8 +58,8 @@ export interface ICorpusFormSubmit {
   import_id_part3?: string
   import_id_part4?: string
   title: string
-  description: string
-  corpus_text?: string | null
+  description?: string | null
```

Comment:
```suggestion
  description?: string
```

Not sure why it needs to be `string | null | undefined` (I can see this is copy/pastaed though).